jll/E/Exodus_jll/Compat.toml

```diff
@@ -1,4 +1,5 @@
 [0]
+Fmt_jll = "9.0.0-1"
```

Comment:
This syntax doesn't make sense (it means from version 9.0.0 to 1), you need something like
```toml
Fmt_jll = "9"
```
But also, if the compat is the same everywhere, you can also have a single section
```toml
["0-8"]
Fmt_jll = "9"
```

Reply:
I went with your second recommendation for a single section. Thanks for the help. This was the first time I had to do anything with the registry directly.